src/appleseed-max-impl/appleseedobjpropsmod/appleseedobjpropsmod.cpp

```diff
@@ -75,7 +75,8 @@ namespace
         ParamIdVisibilitySpecular       = 7,
         ParamIdVisibilitySSS            = 8,
         ParamIdSSSSet                   = 9,
-        ParamIdOptimizeForInstancing    = 10
+        ParamIdOptimizeForInstancing    = 10,
+        paramIdNestedmediumPriority     = 11
```

Comment:
Capitalize *Param* and *Medium*: `ParamIdNestedMediumPriority`

Thinking of it, I think `ParamIdMediumPriority` (without *Nested*) would be better.

Reply:
OK